react/src/components/SessionDetailContent.tsx

```diff
@@ -240,10 +240,7 @@ const SessionDetailContent: React.FC<{
             }
           />
           <Button.Group size="large">
```

Comment:
Button.Group is deprecated since antd version 5.24.0

https://ant.design/changelog#5240